packages/generators/generators/lib/plops/prompts/get-attributes-prompts.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const validateInput = require('../utils/validate-input');
+const validateInputForAttribute = require('../utils/validate-input');
```

Comment:
you will have to import the new validator you created now :)
In term of naming I would go with the same as the file name so validateAttributeInput

Reply:
Whoops. Fixed it